### DISCUSSION TITLE
Renovate only updates Rust's Cargo.lock, but not Cargo.toml

### DISCUSSION THREAD

**Discussion by hongbo-miao** [renovatebot/renovate#32890] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

I noticed from today, Renovate only updates Rust's Cargo.lock, but not Cargo.toml such as

- https://github.com/hongbo-miao/hongbomiao.com/pull/21152
- https://github.com/hongbo-miao/hongbomiao.com/pull/21158

**Comment by rarkins** [comment]
See the "No exact versions unless using equals =" section of https://github.com/hongbo-miao/hongbomiao.com/pull/21152

If you want to see the Cargo.toml to be bumped in sync with Cargo.lock, you'll need rangeStrategy=bump

**Reply by hongbo-miao** [reply, marked as answer]
Thank you @rarkins ! My bad, in Cargo, I originally thought

```
[dependencies]
apache-avro = "0.17.0"
tokio = { version = "1.41.0", features = ["full"] }
```

means pin the version.

Turns out, I have to add extra `=` like

```
[dependencies]
apache-avro = "=0.17.0"
tokio = { version = "=1.41.0", features = ["full"] }
```

This is slightly different with other languages' package manangement.
